docs/1-introduction/1-1-getting-started/1-1-1-hello-window.md

```diff
@@ -98,18 +98,18 @@ resolution you use. Size-wise it will cover 90% of your main monitor.
 glfwWindowHint(GLFW_SCALE_TO_MONITOR, GLFW_FALSE);
 ```
 
-This will tell `GLFW` to not scale the window in any way, should you have setup a specific scaling
-other than 100% on your desktop. That will keep the windowsize at what we set it, without thinking
-about odd fractionals to manually scale the windowsize for any arbitrary scaling on your OS.
+This will tell `GLFW` to not scale the window in any way, should you have set up a specific scaling
+other than 100% on your desktop. That will keep the window size at what we set it, without thinking
+about odd fractional to manually scale the window size for any arbitrary scaling on your OS.
```

Comment:
hmm, isn't the right word `fractions`?

Reply:
I've changed it to `, and lets us
forget about fractional window and pixel scaling.`. I think that's more understandable than what we currently have.